src/main/resources/static/eiffel-rabbit-mq-post-receive-hook.soy

```diff
@@ -15,25 +15,43 @@
     {param value: $config['rabbit.url'] /}
     {param descriptionText: 'RabbitMQ server address to connect to.' /}
     {param errorTexts: $errors ? $errors['rabbit.url'] : null /}
-    {param rows: '4' /}
+    {param rows: '1' /}
   {/call}
 
```

Comment:
Add a field to read port, make sure to mention default value, i.e. will be set to 5672 is left empty. Is it possible to pass default value here so we don't hard code it?